IDEAS/Fluid/PvtCollectors/Validation/PVT1/BaseClasses/package.mo

```diff
@@ -0,0 +1,3 @@
+within IDEAS.Fluid.PVTCollectors.Validation.PVT1;
```

Comment:
I noticed that both PVT1 & PVT2 and PVT_UI & PVT_UN are part of this PR. Please remove PVT1 & PVT2 and take all comments above (for Validation.PVT1) into account for Validation.PVT_UI. I will now continue to Validation.PVT_UI.Electrical